python/parser.js

```diff
@@ -4,16 +4,14 @@ const types = require('./mapping/types')
 const selectors = require('./mapping/selectors')
 
 function checkSettings(cs) {
-    let name = ''
-    let target = `Target`
     if(cs === undefined){
-        return target + '.window()'
+        return 'Target.window()'
     }
-    let element = ''
-    let options = ''
-    element = '.window()'
+    let name = cs.name ? python`${cs.name}, ` : '';
+    let target = cs.webview ? python`Target.webview(${cs.webview})` : 'Target.window()';
```

Comment:
The input to webview is either a boolean or a string. Wouldn't the string need to be enclosed in quotes? And would the boolean need to be capitalized?

Reply:
This is how this python`` magic works. I didn't invent it and I'm not that good at JS to tell the details, but from its behavior, I see it as a filter that is applied to all substitution variables and actually transforms them into pythonic equivalent. So strings get quoted and booleans capitalized.

Reply:
Fair enough.